build-docker.sh

```diff
@@ -1,5 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
The build script is a separate script so that you can build the images locally and then play with them, and as such should not be removed.

Reply:
Also the upcoming production deploy will require it.

Reply:
ok, then I will add the build script back